codegen/tests/simple/app.go

```diff
@@ -0,0 +1,89 @@
+// Code generated by sysl DO NOT EDIT.
+package simple
+
+import (
+	"context"
+
+	"github.com/anz-bank/sysl-go/codegen/tests/deps"
+	"github.com/anz-bank/sysl-go/config"
+	"github.com/anz-bank/sysl-go/core"
+	"github.com/anz-bank/sysl-go/handlerinitialiser"
+)
+
+// HandlerManager for Simple
+type HandlerManager struct {
+	coreCfg             *config.DefaultConfig
+	enabledHandlers     []handlerinitialiser.HandlerInitialiser
+	enabledGrpcHandlers []handlerinitialiser.GrpcHandlerInitialiser
+}
+
+// NewHandlerManager create new HandlerManager for Simple
+func NewHandlerManager(coreCfg *config.DefaultConfig) *HandlerManager {
+	mgr := &HandlerManager{
+		coreCfg:             coreCfg,
+		enabledHandlers:     []handlerinitialiser.HandlerInitialiser{},
+		enabledGrpcHandlers: nil,
+	}
+
+	return mgr
+}
+
+// EnabledHandlers ...
+func (h *HandlerManager) EnabledHandlers() []handlerinitialiser.HandlerInitialiser {
+	return h.enabledHandlers
+}
+
+// EnabledGrpcHandlers ...
+func (h *HandlerManager) EnabledGrpcHandlers() []handlerinitialiser.GrpcHandlerInitialiser {
+	return h.enabledGrpcHandlers
+}
+
+// LibraryConfig ...
+func (h *HandlerManager) LibraryConfig() *config.LibraryConfig {
+	return &h.coreCfg.Library
+}
+
+// LoadServices ...
+func (h *HandlerManager) LoadServices(ctx context.Context) error {
+	bffSI := ServiceInterface{
```

Comment:
bffSI should be replaced with a generic name
```suggestion
	appSI := ServiceInterface{
```

Reply:
Plan to remove it.